src/main/java/fr/esgi/al5_2/Tayarim/TayarimApplication.java

```diff
@@ -16,92 +16,90 @@
 @SpringBootApplication
 public class TayarimApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(TayarimApplication.class, args);
-	}
-
-	// @Bean
-    // public CommandLineRunner demo(ProprietaireRepository repository) {
-    //     return (args) -> {
-
-	// 		// Récupérer tous les propriétaires et les afficher
-    //         List<Proprietaire> proprietaires = repository.findAll();
-	// 		System.out.println(proprietaires.size());
-    //         for (Proprietaire p : proprietaires) {
-    //             System.out.println(p.getId() + " " + p.getNom() + " " + p.getPrenom());
-    //         }
-
-    //         // Exemple: Enregistrer une nouvelle personne
-    //         Proprietaire proprietaire = new Proprietaire("Ferreira", "Mathieu", "test@gmail.com", "0675000495", "password", LocalDateTime.now());
-    //         repository.save(proprietaire);
-
-	// 		// Exemple: Enregistrer une nouvelle personne
-    //         Proprietaire proprietaire2 = new Proprietaire("Ferreira", "Mat", "test@gmail.com", "0675000495", "password", LocalDateTime.now());
-    //         repository.save(proprietaire2);
-
-	// 		// Récupérer tous les propriétaires et les afficher
-    //         proprietaires = repository.findAll();
-	// 		System.out.println(proprietaires.size());
-    //         for (Proprietaire p : proprietaires) {
-    //             System.out.println(p.getId() + " " + p.getNom() + " " + p.getPrenom());
-    //         }
-    //     };
-    // }
+  public static void main(String[] args) {
+    SpringApplication.run(TayarimApplication.class, args);
+  }
+
+  // @Bean
+  // public CommandLineRunner demo(ProprietaireRepository repository) {
+  //     return (args) -> {
+
+  // 		// Récupérer tous les propriétaires et les afficher
+  //         List<Proprietaire> proprietaires = repository.findAll();
+  // 		System.out.println(proprietaires.size());
+  //         for (Proprietaire p : proprietaires) {
+  //             System.out.println(p.getId() + " " + p.getNom() + " " + p.getPrenom());
+  //         }
+
+  //         // Exemple: Enregistrer une nouvelle personne
+  //         Proprietaire proprietaire = new Proprietaire("Ferreira", "Mathieu", "test@gmail.com", "0675000495", "password", LocalDateTime.now());
+  //         repository.save(proprietaire);
+
+  // 		// Exemple: Enregistrer une nouvelle personne
+  //         Proprietaire proprietaire2 = new Proprietaire("Ferreira", "Mat", "test@gmail.com", "0675000495", "password", LocalDateTime.now());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 143).